doc/source/whatsnew/v0.25.0.rst

```diff
@@ -18,10 +18,7 @@ including other versions of pandas.
 
 Other Enhancements
 ^^^^^^^^^^^^^^^^^^
-
--
--
--
+:func:`DataFrame.plot` keywords ``logy``, ``logx`` and ``loglog`` can now accept the value ``sym`` for symlog scaling. (:issue:`24867`)
```

Comment:
There should still be a leading "-" before the entry:
```rst
- :func:`DataFrame.plot` keywords ...
```

Reply:
done! thanks for the review!